content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+Kubernetes metrics. 
+
+Receiving data:
+* [Kubernetes Cluster Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver#example): collects cluster-level metrics and 
+entity events from the [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
+* [Kubernetes Objects Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver):  collects (pull/watch) objects from the 
+Kubernetes API server
+* [Kubelet Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver): pulls node, pod, container, and volume metrics 
+from the API server on a [Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) and sends it down the metric pipeline for further processing
+* [Host Metrics Receiver](https://opentelemetry.io/docs/kubernetes/collector/components/#host-metrics-receiver): scrapes system metrics from hosts that make up the 
+cluster
+
+Processing data:
+* [Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor): adds Kubernetes context, thereby 
+* enabling you to correlate application telemetry with your Kubernetes telemetry 
+* – considered one of the most important components for monitoring Kubernetes 
+* with OpenTelemetry 
+
+You can also use the Kubernetes attributes processor to set custom resource 
+attributes for traces, metrics, and logs using the Kubernetes labels and 
+annotations you’ve added to your pods and namespaces.
+
+There are a few more collector components you can implement to monitor 
+Kubernetes, including Kubernetes-specific ones as well as general-use 
+processors, such as the batch, memory limiter, and resource processors. You can 
+read more about them [here](docs/kubernetes/collector/components/). 
+
+After you’ve configured the components in your collector config file, you need 
+to enable them within the [pipelines](https://opentelemetry.io/docs/collector/configuration/#pipelines) section. A data pipeline enables you to [collect](https://opentelemetry.io/docs/collector/configuration/#receivers), [process](https://opentelemetry.io/docs/collector/configuration/#processors), 
+and route data from any source [to one destination or more](https://opentelemetry.io/docs/collector/configuration/#exporters). 
+
+## Pros & Cons
+However you decide to use OTel to gather Prometheus metrics, ultimately what is 
+right for your organization depends on your business needs. Using the OTel 
+components discussed above, you could convert all your metrics into the 
+Prometheus format, or you could convert your Prometheus metrics into OTLP. 
+Although Prometheus itself was not built for long-term data storage and presents 
+scaling challenges, there are tools such as [Grafana](https://grafana.com/), 
```

Comment:
Do you perhaps mean Grafana Mimir, as opposed to Grafana? Grafana Mimir is Grafana Labs' horizontally scalable/long term storage Prometheus compatible product, and is comparable to Thanos and Cortex (Grafana Mimir was initially forked from Cortex):
```suggestion
scaling challenges, there are products such as [Grafana Mimir](https://grafana.com/oss/mimir/), 
```